### PR TITLE
feat(alerts): Remove duplicate styles, useless translations

### PR DESCRIPTION
**Pull request by scttcper**
- Stop using a translated string as the status color
- Remove useless duration translation
- Add types to StatusIndicator

### CODE REVIEW

File: static/app/views/alerts/rules/metric/details/metricHistory.tsx [hunk 1]
```diff
@@ -77,58 +65,59 @@ function MetricAlertActivity({organization, incident}: MetricAlertActivityProps)
...
+            tooltipTitle={t('Status: %s', triggerStatus)}
+          />
+        )}
         <Link
           to={{
             pathname: alertDetailsLink(organization, incident),
             query: {alert: incident.identifier},
           }}
         >
-          {tct('#[id]', {id: incident.identifier})}
```
Comment (scttcper) [inline static/app/views/alerts/rules/metric/details/metricHistory.tsx:92]: useless translation

File: static/app/views/alerts/rules/metric/details/metricHistory.tsx [hunk 2]
```diff
@@ -63,12 +57,6 @@ function MetricAlertActivity({organization, incident}: MetricAlertActivityProps)
     nextActivity ? moment(nextActivity.dateCreated) : moment()
   ).diff(moment(triggeredActivity.dateCreated), 'milliseconds');
 
-  const threshold =
-    activityDuration !== null &&
-    tct('[duration]', {
```
Comment (scttcper) [inline static/app/views/alerts/rules/metric/details/metricHistory.tsx:68]: useless translation

File: static/app/views/alerts/rules/metric/details/metricHistory.tsx [hunk 3]
```diff
@@ -77,58 +65,59 @@ function MetricAlertActivity({organization, incident}: MetricAlertActivityProps)
     .map(trig => trig.alertThreshold);
 
   return (
-    <ErrorBoundary>
-      <Title data-test-id="alert-title">
-        <StatusIndicator
-          status={currentTrigger.toLocaleLowerCase()}
```
Comment (scttcper) [inline static/app/views/alerts/rules/metric/details/metricHistory.tsx:83]: translated string as a status/enum

File: static/app/views/alerts/rules/metric/details/metricHistory.tsx [hunk 4]
```diff
@@ -43,92 +32,85 @@ function MetricAlertActivity({organization, incident}: MetricAlertActivityProps)
...
     activity => activity.value === `${IncidentStatus.WARNING}`
   );
 
-  const triggeredActivity = criticalActivity.length
-    ? criticalActivity[0]
-    : warningActivity[0];
-  const currentTrigger = getTriggerName(triggeredActivity.value);
+  const triggeredActivity: ActivityType = criticalActivity
+    ? criticalActivity!
+    : warningActivity!;
```
Comment (billyvg) [inline static/app/views/alerts/rules/metric/details/metricHistory.tsx:44]: Is it ever possible that there is no triggered activity?

File: static/app/views/alerts/rules/metric/details/metricHistory.tsx [hunk 5]
```diff
@@ -43,92 +32,85 @@ function MetricAlertActivity({organization, incident}: MetricAlertActivityProps)
...
+  const isCritical = Number(triggeredActivity.value) === IncidentStatus.CRITICAL;
 
+  // Find duration by looking at the difference between the preiovus and current activity timestamp
```
Comment (billyvg) [inline static/app/views/alerts/rules/metric/details/metricHistory.tsx:47]:
```suggestion
  // Find duration by looking at the difference between the previous and current activity timestamp
```

**Review by billyvg — APPROVED** [review]

File: static/app/views/alerts/rules/metric/details/metricHistory.tsx (hunk 4 above)
Comment (scttcper) [reply static/app/views/alerts/rules/metric/details/metricHistory.tsx:44]: I don't think so. it was using `.filter` with `criticalActivity[0]` before which our typescript just isn't setup to check if its undefined

**Pull request merged**